source/docs/03-metadata.xml

```diff
@@ -718,11 +718,11 @@
                <p>The IFLA’s FRBR model distinguishes four levels of abstraction, or entities:</p>
                <list type="gloss">
                   <label>Work</label>
-                  <item>FRBR defines a work as a "distinct intellectual or artistic creation", an abstract entity because there is no single material object one can point to as the work.</item>
+                  <item>FRBR defines a work as a <quote>distinct intellectual or artistic creation</quote>, an abstract entity because there is no single material object one can point to as the work.</item>
```

Comment:
Since these are inline-quotations, `q` should be used instead of block-level `quote`?

Reply:
Are you confusing [MEI](https://music-encoding.org/guidelines/v5/elements/quote.html) with [TEI](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-quote.html)?

Reply:
Yes, indeed, just recognized that TEI does not make this assumption.

I guess I got confused by #1498 where it says that `q` should be used... Maybe you can add `quote` there, too, and give some hint when which of them should be used? Could be helpful for those who will work on it as "good first issue" ;)